assets/js/components/setup/CompatibilityChecks/CompatibilityErrorNotice.js

```diff
@@ -172,32 +175,13 @@ export default function CompatibilityErrorNotice( { error } ) {
 			);
 		case ERROR_AMP_CDN_RESTRICTED:
 			return (
-				<p
-					dangerouslySetInnerHTML={ sanitizeHTML(
-						`
-						${ __(
-							'Looks like the AMP CDN is restricted in your region, which could interfere with setup on the Site Kit service.',
-							'google-site-kit'
-						) }
-						<br/>
-						${
-							sprintf(
-								/* translators: %1$s: Support Forum URL, %2$s: Error message */ // eslint-disable-line indent
-								__(
-									'To get more help, ask a question on our <a href="%1$s">support forum</a> and include the text of the original error message: %2$s',
-									'google-site-kit'
-								), // eslint-disable-line indent
-								'https://wordpress.org/support/plugin/google-site-kit/', // eslint-disable-line indent
-								`<br/>${ error }` // eslint-disable-line indent
-							) /* eslint-disable-line indent */
-						}
-						`,
-						{
-							ALLOWED_TAGS: [ 'a', 'br' ],
-							ALLOWED_ATTR: [ 'href' ],
-						}
-					) }
-				/>
+				<p>
+					{ __(
+						'Looks like the AMP CDN is restricted in your region, which could interfere with setup on the Site Kit service.',
+						'google-site-kit'
+					) }{ ' ' }
```

Comment:
There is mixed feelings and prior arts about adding the space like this. The general consensus is to use `createInterpolateElement` to make the space localizeable but I don't have any strong feeling about it here. What do you think? 

Some previous discussion regarding it can be found in #5649 and #5646

Reply:
Updated; thanks for the suggestion.